9.mAP/notebooks/1.aggregate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>The comment above does not reflect what is occurring in this block. Consider removing comments <code>lines: 1-5</code> in order to improve readability </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>